_data/sites.json

```
@@ -3961,6 +3961,16 @@
         ]
     },
 
+    {
+        "name": "discovery+",
+        "url": "https://support.discoveryplus.com/hc/en-gb/articles/360018959458-How-do-I-delete-my-account-",
```

```suggestion
        "url": "https://privacyportal.onetrust.com/webform/1b21e05d-c206-4e0b-970e-2d73a23e42e8/5b504afe-dd1e-47f6-9af6-56144531eced",
```

Hey @WebSnke, if you look at Discovery's [privacy notice](https://corporate.discovery.com/privacy-policy/) updated in 2023, you'll see they provide a link to an external [web form](https://privacyportal.onetrust.com/webform/1b21e05d-c206-4e0b-970e-2d73a23e42e8/5b504afe-dd1e-47f6-9af6-56144531eced) to be filled out which provides a direct way for users to create a request.